Gemfile

```diff
@@ -0,0 +1,60 @@
+source 'https://rubygems.org'
+
+
```

Comment:
I know there are dependencies that require us to consider the order of the gems listed, but in order to better group our gemfile, should we have our lists more clearly separated?
i.e.
production do
development do
testing and development do
testing do